agentos/registry.py

```diff
@@ -83,6 +84,54 @@ def from_repo(
         ), "YAML is the only registry file format supported currently"
         return cls.from_yaml(repo.get_local_repo_dir(version) / file_path)
 
+    @classmethod
+    def from_repo_inferred(
+        cls,
+        repo: "Repo",
+        py_file_suffixes: Sequence[str] = [".py", ".python"],
```

Comment:
It might be good to [avoid mutables-as-default-args](https://stackoverflow.com/questions/9158294/good-uses-for-mutable-function-argument-default-values) unless we're taking advantage of their somewhat-surprising behavior. 

Maybe default this to `None` and then assign it this list in the body if it is, in fact, `None`.

Reply:
Good catch. I changed it to:
```python
        py_file_suffixes: Tuple[str] = (".py", ".python"),
```
since tuple is immutable.

Reply:
👍